setup/fs

```diff
@@ -119,7 +119,7 @@ mount_fs() {
 	for dev in $bdevs
 	do
 		local mnt=/fs/$(basename $dev)
-		is_mount_on_root $dev || log_cmd mkdir -p $mnt
+		log_cmd mkdir -p $mnt
```

Comment:
thanks, can you help add Fixes tag in change log to show which early commit is fixed?